server.py

```diff
@@ -270,6 +301,10 @@ def add(username):
         TheGrid.AllPlayers[username] = Player(start_position[0], start_position[1], start_position[2],
                                               username, TheGrid.Speed, [], [], [])
         TheGrid.UsersNum += 1
+        converted = []
+        for i in TheGrid.boosters:
```

Comment:
You can use i for index variables (but if you can give a logical name then it's better) but for other loops use some better names like booster here